docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,353 @@ Review important information about the {kib} 8.x releases.
+
+
+[float]
+[[enhancement-v8.16.0]]
+=== Enhancements
+Alerting::
+* Allow users to select template while adding a case action in the rule ({kibana-pull}190701[#190701]).
+Dashboards and visualizations::
+* Adds compressed style for dashboard controls ({kibana-pull}190636[#190636]).
+* Adds the ability to duplicate a managed dashboard from its `managed` badge ({kibana-pull}189404[#189404]).
+* Syncs the cursor for time series charts powered by ES|QL ({kibana-pull}192837[#192837]).
+* Gauge and metric Lens visualizations are no longer experimental ({kibana-pull}192359[#192359]).
+* Sets gauge default palette to "temperature" in *Lens* ({kibana-pull}191853[#191853]).
+* Improves metric styling options in *Lens* ({kibana-pull}186929[#186929]).
+* Supports fuzzy search on field pickers and field lists in *Lens* ({kibana-pull}186894[#186894]).
+Data ingestion and Fleet::
+* Update max supported package version ({kibana-pull}196551[#196551]).
+* Adds additional columns to Agent Logs UI ({kibana-pull}192262[#192262]).
+* Show `+build` versions for Elastic Agent upgrades ({kibana-pull}192171[#192171]).
+* Added format parameter to `agent_policies` APIs ({kibana-pull}191811[#191811]).
+* Adds toggles for `agent.monitoring.http.enabled` and `agent.monitoring.http.buffer.enabled` to agent policy advanced settings ({kibana-pull}190984[#190984]).
+* Support integration policies without agent policy references (aka orphaned integration policies) ({kibana-pull}190649[#190649]).
+* Changed the UX of the Edit Integration Policy page to update agent policies ({kibana-pull}190583[#190583]).
+* Allow `traces` to be added to the `monitoring_enabled` array in Agent policies ({kibana-pull}189908[#189908]).
+* Create task that periodically unenrolls inactive agents ({kibana-pull}189861[#189861]).
+* Adds setup technology selector to add integration page ({kibana-pull}189612[#189612]).
+* Support integration-level outputs ({kibana-pull}189125[#189125]).
+Discover::
+* Renames the Documents tab to Results in ES|QL mode ({kibana-pull}197833[#197833]).
+* Adds a cluster details tab for CCS data sources when inspecting requests in ES|QL mode ({kibana-pull}195373[#195373]).
+* Adds the query time to the list of statistics when inspecting requests in ES|QL mode ({kibana-pull}194806[#194806]).
+* Changes the auto-focus to be on the ES|QL editor when loading the page ({kibana-pull}193800[#193800]).
+* Updates the autocomplete behavior for `SORT` to be in line with other field-list-based experiences like `KEEP` in ES|QL queries ({kibana-pull}193595[#193595]).
+* Adds the ability to expand the height of various sections in the Edit ES|QL visualization flyout ({kibana-pull}193453[#193453]).
+* Adds `all (*)` to the list of suggestions for `COUNT` functions in ES|QL queries ({kibana-pull}192205[#192205]).
+* Enhances the inline documentation experience in ES|QL mode ({kibana-pull}192156[#192156]).
```

Comment:
This is my fault as I used the enhancement label but can we move this into the features section? Many users have complained about it so I would like to advertise it more